frank-doc-doclet/src/main/java/org/frankframework/frankdoc/ElementGroupManager.java

```diff
@@ -55,8 +54,8 @@ String addGroup(Set<ElementRole.Key> key) {
 		String roleName = getRoleName(key);
 		List<Set<ElementRole.Key>> shared = genericGroupKeyToSeq.keySet().stream()
 				.filter(rs -> getRoleName(rs).equals(roleName))
-				.collect(Collectors.toList());
-		int seq = shared.stream().map(genericGroupKeyToSeq::get).collect(Collectors.maxBy(Integer::compare)).orElse(0) + 1;
+				.toList();
+		int seq = shared.stream().map(genericGroupKeyToSeq::get).max(Integer::compare).orElse(0) + 1;
```

Comment:
Dat kan gecombineerd worden denk ik:

```suggestion
int seq = genericGroupKeyToSeq.keySet().stream()
			.filter(rs -> getRoleName(rs).equals(roleName))
			.map(genericGroupKeyToSeq::get)
			.max(Integer::compare).orElse(0) + 1;
```